### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Holo314**
Solved the problem of expressions inside function parameters not count well by counting parentheses and automatically wrapping parameters of a function with extra layer of parentheses (e.g. `if(1*1,4,5)` -> `if((1*1),(4),(5))`  and `if((1*1),4,5)` -> `if(((1*1)),(4),(5))`)

### CODE REVIEW

**Comment by Holo314** [comment]
Should resolve #32

**Review by notKamui — CHANGES_REQUESTED** [review]
Everything seems good to me ; just a few changes for documentation sake and it's good to go

Thanks a lot for your contribution to this project (again !)

File: README.md [hunk 1]
```diff
@@ -173,6 +173,8 @@ operator to
 "(2+3)(6+4)".keval() == "(2+3)*(6+4)".keval()
 ```
 
+In addition the symbols `(`,`)`,`,` are reserved and trying to create operator using one of those symbols will result with an exception.
+
```
Comment (notKamui) [inline README.md:177]: good catch

**Review by notKamui — APPROVED** [review]
Thank you ! Closing this PR

**Pull request merged**